globus_cli/commands/task/list.py

```diff
@@ -6,16 +6,141 @@
 from globus_cli.services.transfer import print_json_from_iterator, get_client
 
 
+class ISOTimeType(click.ParamType):
+    """
+    Enforces time inputs to be in either YYYY-MM-DD or YYYY-MM-DD HH:MM:SS
+    And that each value is a valid int that falls in the correct range
+    """
+
+    name = "ISO TIME"
```

Comment:
Can we make this `ISO_TIME`? The Type name will sometimes show up as a metavar in help output, and we've had a past complaint that spaces in metavars were confusing.
(I know it's silly, but humor me? :smile: )